### ISSUE TITLE
Customization of DeclarationOrder

### ISSUE THREAD

**Issue by robcolburn** [causes/scss-lint#333]
I really like a rule like `DeclarationOrder`, except when I need to break it… a lot.  This mostly happens with breakpoints.

```scss
.cat {
  width: 1000px;
  @include breakpoint($handheld) {
    width: 200px;
  }
}
```

I suppose we could say I'm looking to relax it when using a `@context` mixin.

**Comment by lencioni** [comment]
Thanks for the suggestion. However, according to [the documentation](https://github.com/causes/scss-lint/blob/master/lib/scss_lint/linter/README.md#declarationorder), I think this should already be the case:

> Rule sets should be ordered as follows: @extend declarations, @include declarations without inner @content, properties, @include declarations with inner @content, then nested rule sets.

...

> Placing @include declarations without inner @content before properties serves to group them with @extend declarations and provides the opportunity to overwrite them later in the rule set.

> @includes with inner @content often involve @media rules that rely on the cascade or nested rule sets, which justifies their inclusion after regular properties.

And, I think we have specs that verify this behavior as well: https://github.com/causes/scss-lint/blob/4b83c6c248e27f9896efc57621b23855c883510d/spec/scss_lint/linter/declaration_order_spec.rb#L168

Is there a specific warning that you are seeing?

**Comment by lencioni** [comment]
Also, what version of scss-lint are you using?

**Comment by robcolburn** [comment]
Ah, so my problem is even more of an edge case.  I want to be able add breakpoints (include w/ content) after nested rules.

```scss
.cat {
  // Default/Mobile case
  &:before {
    content: 'W';
  }

  // Specific breakpoints
  @include breakpoint($desktop) {
    &:before {
      content: 'Whiskers';
    }
  }
}
```

This example could easily be refactored, but I have more complicated cases.  Again, love everything else about DeclarationOrder.

**Comment by robcolburn** [comment]
I've decided in my specific cases, I'll just add inline disable rules, not too bad of an experience.
